app.js

```diff
@@ -12,6 +12,13 @@ app.get('/api/topics', getTopics);
 
 app.get('/api/articles/:article_id', getArticles);
 
+app.use((err, req, res, next) => {
+    if(err.message === "Bad Request" || err.code === '22P02'){
+        res.status(400).send({msg:"Bad Request"})
+    }
+    else res.status(404).send({msg:"Not Found"})
+})
+
 app.use((err, req, res, next) => {
     res.status(500).send({msg:"Server is unable to access data at the moment"})
 })
```

Comment:
The logic here is good, however these should be 2 separate error handlers - one for PSQL errors and one for custom errors (400). You can make this dynamic by sending back the existing err.status & err.msg.